external/storm-kafka/pom.xml

```diff
@@ -129,7 +129,7 @@
             <groupId>org.apache.storm</groupId>
             <artifactId>storm-core</artifactId>
             <version>${project.version}</version>
-            <scope>provided</scope>
+            <!--scope>provided</scope-->
```

Comment:
good catch @HeartSaVioR . I had to make this change for in-IDE testing. I will revert this.